permissions/pom-archetypes.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "archetypes"
+github: "jenkinsci/archetypes"
```

Comment:
this is deployed to maven central why are you adding it here?

Reply:
Oic, yeah, 🤦🏻 me

Gonna remove it